### PR TITLE
Add "@tailwindcssinjs/macro"

### PR DESCRIPTION
**Pull request by Arthie**
Hello!
I added "@tailwindcssinjs/macro" to the tools section.

Package description:
@tailwindcssinjs/marco is a babel macro that transforms Tailwind classes into css object styles. These css object styles can be used with your favorite css-in-js library.

Full disclosure: I am the author of this package.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -144,6 +144,7 @@
 - 🌍🔧 [TailwindCSS Palette Generator](https://tailwindcss-palette-generator.coderello.com/) - Palette generator for Tailwind CSS (online).
 - 💼 [@nuxtjs/tailwindcss](https://github.com/nuxt-community/tailwindcss-module) - Tailwind CSS module for NuxtJS with PurgeCSS and modern CSS (preset env 1).
 - 💼 [preact-cli-tailwind](https://github.com/agneym/preact-cli-tailwind) - Adds Tailwind CSS module as PostCSS plugin and sets up PurgeCSS in production for PreactJS CLI Projects.
+- 💼🔧 [@tailwindcssinjs/macro](https://github.com/Arthie/tailwindcssinjs) - Babel macro that transforms Tailwind CSS classes into css object styles for css-in-js libraries.
```
Comment (hawezo) [inline readme.md:147]: That looks really nice! May you correct this before I merge, though?

```suggestion
+ 💼🔧 [@tailwindcssinjs/macro](https://github.com/Arthie/tailwindcssinjs) - Babel macro that transforms Tailwind CSS classes into objects for CSS-in-JS libraries.
```

**Comment by joshmanders** [comment]
Great work @Arthie. I starred the repo and will most likely start using it myself. As soon as you resolve @hawezo's correction we'll merge in.

File: readme.md (hunk 1 above)
Comment (Arthie) [reply readme.md]: Ok fixed!

**Comment by Arthie** [comment]
@joshmanders Thank you! I resolved the issue.

**Pull request merged**